### DISCUSSION TITLE
Custom Pins/Label on Website

### DISCUSSION THREAD

**Discussion by indo2042** [LBPUnion/ProjectLighthouse#691] [Ideas]
Instance administrators and moderators have their usernames displayed as a different color than other, normal users.

![image](https://user-images.githubusercontent.com/114881121/221438440-45eeef17-950e-47dd-ad10-5cacb30d6f19.png)

It would be nice to have an option, possibly in the Admin Panel, to add custom pins/labels and award them to users. Using Beacon as an example, we could award our beta testers (pre-public beta launch) a special pin on the website; we could also award competition winners one as well.

**Comment by sudokoko** [comment]
Hey there, thanks for your suggestion! I'll pass this on to the R&D team.

**Reply by indo2042** [reply]
Silence fed.
